cg/meta/orders/api.py

```diff
@@ -57,11 +58,15 @@ def _get_submit_handler(project: OrderType, lims: LimsAPI, status: Store) -> Sub
 class OrdersAPI:
     """Orders API for accepting new samples into the system."""
 
-    def __init__(self, lims: LimsAPI, status: Store, osticket: OsTicket):
+    def __init__(
+        self, lims: LimsAPI, status: Store, osticket: OsTicket, freshdesk_client: FreshdeskClient
+    ):
         super().__init__()
         self.lims = lims
         self.status = status
-        self.ticket_handler: TicketHandler = TicketHandler(osticket_api=osticket, status_db=status)
+        self.ticket_handler: TicketHandler = TicketHandler(
```

Comment:
The ticket handler should preferably be passed into the `OrdersAPI` class directly as a dependency instead of being instantiated here, but that is a larger refactoring. Feel free to consider it out of scope.

Just want to make you aware of the existing design issues.